app/src/main/java/com/sunasterisk/loltournaments/ultils/ImageViewExtensions.kt

```diff
@@ -0,0 +1,11 @@
+package com.sunasterisk.loltournaments.ultils
+
+import android.widget.ImageView
+import androidx.databinding.BindingAdapter
+import com.bumptech.glide.Glide
+import com.sunasterisk.loltournaments.R
+
+@BindingAdapter("android:loadImage")
```

Comment:
a nghĩ e nên đặt 1 cái tên khác thay cho "android:loadImage" ,  vì đặt như này khi đọc lại code dễ nhầm cái attr loadImage này do android cung cấp
```suggestion
@BindingAdapter("loadImageUrl")
```